docs/appendices/release-notes/5.8.0.rst

```diff
@@ -44,7 +44,9 @@ Version 5.8.0 - Unreleased
 Breaking Changes
 ================
 
-None
+- Removed ``network`` column from :ref:`sys.nodes <sys-nodes>` table. The column
```

Comment:
@smu do you agree on a `breaking change` for this removal? It was not in use anyway (only 0s were returned) and clearly marked as deprecated since `2.3.0`.

Reply:
Yes